schema/event_mapping_entity.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$udmi_version": "1.4.0",
+  "title": "Mapping Event Entity",
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "code": {
+      "description": "",
+      "type": "string"
+    },
+    "type": {
+      "description": "",
+      "type": "string"
+    },
+    "cloud_device_id": {
+      "description": "",
+      "type": "string"
+    },
+    "connections": {
```

Comment:
should not be empty

Reply:
happy to fix this, but not sure I understand what the guidelines are. For example I see empty schema maps checked in at https://github.com/grafnu/udmi/blob/7755958bf6801557929a0955cf30b008813f1c49/schema/building_translation.json#L13 and in other places. 

Do the automated tests need to be updated to reflect this guideline? Am I missing or mis-reading the extant code?

Reply:
Those are just preliminary sketches of things, but we should figure out more what they are at this stage.